Test/QueryBuilderSpec.hs

```diff
@@ -141,6 +185,46 @@ tests = do
                     (toSQL theQuery) `shouldBe` ("SELECT posts.id, posts.title, posts.external_url, posts.created_at, posts.public, posts.created_by, posts.category_id FROM posts WHERE posts.category_id IS ?", [Plain "null"])
 
 
+        describe "filterWhereIdIn" do
+            it "should produce a SQL with a WHERE condition" do
+                let theValues :: [Id Post] = ["b80e37a8-41d4-4731-b050-a716879ef1d1", "629b7ee0-3675-4b02-ba3e-cdbd7b513553"]
+                let theQuery = query @Post
+                        |> filterWhereIdIn theValues
```

Comment:
Would `filterWhereIn (#id, theValues)` also still work here? We use `filterWhereIn` quite a lot in our IHP apps

Reply:
Yes, as long as the "id" field isn't virtual and actually corresponds to a column named "id" in the database, this should work exactly as before.
I'll add a unit-test to make sure this is the case.